lib/src/xdg_camera_portal.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:io';
+
+import 'package:dbus/dbus.dart';
+
+import 'xdg_portal_request.dart';
+
+/// Camera portal.
+class XdgCameraPortal {
+  final DBusRemoteObject _object;
+  final String Function() _generateToken;
+
+  XdgCameraPortal(this._object, this._generateToken);
+
+  /// Check to access camera.
+  Future<void> accessCamera() async {
```

Comment:
I think this should be a stream, as the camera access can be closed by the portal later.